tests/compile/test_full_graph.py

```diff
@@ -107,6 +109,18 @@ def test_full_graph(
         (CompilationConfig(level=CompilationLevel.PIECEWISE,
                            debug_dump_path=tempfile.gettempdir()),
          ("facebook/opt-125m", {})),
+    ] + [
+        # graph inductor partition
+        (
+            CompilationConfig(
+                level=CompilationLevel.PIECEWISE,
+                # inductor graph partition uses
+                # torch._C.Tag.cudagraph_unsafe to specify splitting ops
+                use_inductor_graph_partition=True,
+                cudagraph_mode=CUDAGraphMode.PIECEWISE,
+                compile_sizes=[1, 2]),
+            model) for model in models_list(all=False)
+        if is_torch_equal_or_newer("2.9.0.dev")
```

Comment:
Also can we add a test where `enable_attn_fusion` is on (as well as `enable_noop` and `quant_fp8`) to make sure that works? And we can capture the logs and check the "Fused quant onto <n> attention nodes" is present

Reply:
Yes added